galaxy.yml

```diff
@@ -3,7 +3,7 @@
 
 namespace: infra
 name: convert2rhel
-version: 0.1.0
+version: "0.1.0"
```

Comment:
Do we want to start with 1.0.0 for this one, or, change the pre-commit hook to allow lower versions?

Reply:
yea I can change it to 1.0.0 if you want